app/javascript/components/enrollment/steps/Address.js

```diff
@@ -14,6 +14,7 @@ class Address extends React.Component {
       current: { ...this.props.currentState },
       errors: {},
       modified: {},
+      isEditMode: window.location.href.includes('edit'),
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9